src/components/charts/Charts.tsx

```diff
@@ -17,10 +17,9 @@ export default function Charts() {
   const [yAxisSelection, setYAxis] = useState(AggregationFactor.country);
   const isMobileDeviceOrTablet = useMediaQuery({ maxWidth: mobileDeviceOrTabletWidth })
   const [state, dispatch] = useContext(AppContext);
-  const { filters, showAnalyzePopup } = state;
-  const initState = [] as AggregatedRecord[];
+  const { showAnalyzePopup } = state;
   const [showModal, toggleModal] = useState(true);
-  const [records, setRecords] = useState(initState);
+  const records = state.analyze.metaAnalyzedRecords;

```

Comment:
do we even need records here? if it's getting pulled in the references tab anyways

Reply:
we pass records into the bar chart

Reply:
yeah whoops I'm dumb, I misread this as like individual records rather than meta analyzed